src/utils.ts

```diff
@@ -1,4 +1,5 @@
 import { Answer, ChosenAnswer, ID } from '../src/types';
+import { isString, trimStart, trimEnd } from 'lodash';
 
```

Comment:
I pulled in lodash initially to make the transition easier, but in my components I've been attempting to replace it with built in methods ([this](https://gist.github.com/Kris-CDA/9393c6b0c61185a4c4d9426d5dd3a15e) has been a helpful guide.) If we end up having to need it that's okay, but I like the idea of making the bundle size smaller for the consuming application.